tamr_unify_client/mastering/published_cluster/configuration_builder.py

```diff
@@ -0,0 +1,48 @@
+import copy
+
+from tamr_unify_client.mastering.published_cluster.configuration import (
+    PublishedClustersConfiguration,
+)
+
+
+class PublishedClustersConfigurationBuilder:
```

Comment:
we should place this builder in the same file as its built type (`PublishedClustersConfiguration`).